internal/pkg/cli/env_init.go

```diff
@@ -478,6 +482,9 @@ https://aws.amazon.com/premiumsupport/knowledge-center/ecs-pull-container-api-er
 			}
 			return fmt.Errorf("select private subnets: %w", err)
 		}
+		if len(privateSubnets) == 1 {
```

Comment:
If two public subnets are for LB, why can't they use one private subnet?

Reply:
Good question. @efekarakus and I discussed it here: https://github.com/aws/copilot-cli/pull/2356#discussion_r636345844